test/acceptance/specs/BaseSpec.scala

```diff
@@ -38,6 +39,9 @@ trait BaseSpec
     with BrowserDriver
     with Eventually {
 
+  implicit override val patienceConfig: PatienceConfig =
```

Comment:
may be worth extracting this to its own trait, à la `IntegrationPatience`?  then it could be mixed into just the problem suite rather than applied across the board